spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -697,6 +697,7 @@ The method `BeanManager.getInterceptionFactory()` returns an `InterceptionFactor
 <T> InterceptionFactory<T> createInterceptionFactory(CreationalContext<T> ctx, Class<T> clazz);
 ----
 
+If type parameter `T` is not a class non-portable behavior results.
```

Comment:
@johnament abstract class is not a problem IMO. It's like applying an interceptor binding on abstract class method.